### ISSUE TITLE
How can I change design of a site page?

### ISSUE THREAD

**Issue by MarioZagreb** [SharePoint/sp-dev-docs#7989]
### What type of issue is this?

Question

### What SharePoint development model, framework, SDK or API is this about?

💥 SharePoint Framework

### Target SharePoint environment

SharePoint Online

### What browser(s) / client(s) have you tested

- [ ] 💥 Internet Explorer
- [ ] 💥 Microsoft Edge
- [X] 💥 Google Chrome
- [X] 💥 FireFox
- [ ] 💥 Safari
- [ ] mobile (iOS/iPadOS)
- [ ] mobile (Android)
- [ ] not applicable
- [ ] other (enter in the "Additional environment details" area below)

### Issue description

I am creating a custom web parts (mostly different types of forms for users to input different kind of data) and putting them to SP site pages. I need to change the background of that pages (everything in the web part background and around it), but I am having trouble doing and don't know where to start. I tried adding background colors or images, but I don't know in which div should I put that for thing to work on different browsers or screen sizes.

How can I do that so that it works on different windows sizes and different browsers?
I know you can change the theme of the entire site, how can I do that programmatically (it can be for single page or multiple pages, or entire site)?

Any help appreciated.

**Comment by AJIXuMuK** [comment]
@MarioZagreb - could you please explain your scenario?
Why do you need to change the theme of entire site programmatically for a web part?

**Comment by MarioZagreb** [comment]
No, I don't need to change the theme for entire site, I need to change design of site pages and also a page of a document library view, so I was thinking that it is easier to change the theme to the entire site.

My question actually is: How can I change the background of a site page, i.e. I see that each site page has div with the id "spPageCanvasContent", which is where all content of the page is located. I want to change the design of everything in that element.

**Comment by AJIXuMuK** [comment]
@MarioZagreb  - there is no supported way to do such things.
You can definitely apply theme for the whole size (branded colors) but not what you're asking for.
